### PR TITLE
Adding Sierisimo to Kotlin mentors

### PR DESCRIPTION
**Pull request by sierisimo**
I'm getting started with the mentorship and this is included as an step to be listed :)

### CODE REVIEW

File: mentors/kotlin/s.json [hunk 1]
```diff
@@ -6,5 +6,13 @@
...
+        "github_username": "sierisimo",
+        "name": "Sinuhe Jaime",
+        "link_text": null,
+        "link_url": null,
+        "avatar_url": null,
+        "bio": "I'm a geek doing a lot of things, including Kotlin. I've using the language for 3 years now but still learning everyday"
     }
-]
\ No newline at end of file
+]
```
Comment (rpottsoh) [inline mentors/kotlin/s.json]: Unrelated change.

**Comment by rpottsoh** [comment]
To whomever merges this PR, please squash if more than 1 commit at time of merging.

**Comment by sierisimo** [comment]
That's weird. I edited the JSON directly on GitHub. Didn't add that by
myself...
